server/src/main/java/com/cloud/api/query/dao/SnapshotJoinDaoImpl.java

```diff
@@ -258,4 +263,16 @@ public List<SnapshotJoinVO> findByDistinctIds(Long zoneId, Long... ids) {
         sc.setParameters("idsIN", ids);
         return searchIncludingRemoved(sc, searchFilter, null, false);
     }
+
+    public List<SnapshotJoinVO> listBySnapshotIdAndZoneId(Long zoneId, Long id) {
```

Comment:
the name suggests
```suggestion
    public List<SnapshotJoinVO> listBySnapshotIdAndZoneId(Long id, Long zoneId) {
```
maybe change the name or the param order?